FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -383,8 +383,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </handNote>    </handDesc>
                            <additions>
                                <list>
-                                   <item xml:id="e1"><locus target="#131r"></locus><desc type="StampExlibris"><foreign xml:lang="la">
+                                   <item xml:id="e1"><locus target="#1r"></locus><desc>"A. D. 681"</desc></item>
```

Comment:
I doubt that these additions belong to the first codicological unit (msPart p1); I would say they were inserted when the manuscript reached its present shape, already in the library, and thus belong to the physical description of the entire manuscript and not this part

When describing additions we must be careful as to which layer they belong to. Unless it is clear that they were added to a particular part (as in this case, actually part 3 are additions to part 2, even in the end we decided to describe differently) it is safer to leave them with the general description, before the parts begin.